sphinx/source/docs/user_guide/styling.rst

```diff
@@ -621,18 +621,29 @@ for every element of the list:
 Location
 ~~~~~~~~
 
-The location of the legend labels is controlled by the ``orientation``
-property. Valid values for this property are:
-
-``"top_right"``
+The location of the legend labels is controlled by the ``location``
+property. Valid values for this property are either:
 
 ``"top_left"``
 
-``"bottom_left"``
+``"top_center"``
+
+``"top_right"`` (the default)
+
+``"right_center"``
 
 ``"bottom_right"``
 
-The default location is ``"top_right"``.
+``"bottom_center"``
+
+``"bottom_left"``
+
+``"left_center"``
+
+``"center"``
+
+or a ``(x, y)`` tuple indicating an absolute location (measured from
+the bottom-left corner).
```

Comment:
It would be best to try and make all the necessary docs/examples support updates in this PR.